test_ir.py

```diff
@@ -37,5 +37,68 @@ class DisplaySystem(jkt.KernelTests):
     {'code': '1:3', 'mime': {'text/plain':'[1] 1 2 3'}},
     ]
 
+class AbstractIRKernel(jkt.KernelTests):
```

Comment:
I prefer to keep JKT at the simple common layer of unittest, but you've seen how little code it is to start a kernel, so if you want to make a pytest fixture for it, I'm fine with that. We could add it to JKT as a module.